spec/system/search/display_comments_search_spec.rb

```diff
@@ -3,13 +3,12 @@
 RSpec.describe "Display articles search spec", type: :system, js: true, elasticsearch: "FeedContent" do
   it "returns correct results for a search" do
     query = "<marquee='alert(document.cookie)'>XSS"
-    found_comment = create(:comment, body_markdown: query)
-    index_documents_for_search_class([found_comment], Search::FeedContent)
+    create(:comment, body_markdown: query)
 
     url_encoded_query = CGI.escape(query)
     visit "/search?q=#{url_encoded_query}&filters=class_name:Comment"
 
     expect(page.find(".crayons-story__snippet")["innerHTML"])
-      .to eq("…&lt;<mark>marquee</mark>='<mark>alert</mark>(<mark>document.cookie</mark>)'&gt;<mark>XSS</mark>…")
+      .to eq("…<mark>marquee</mark>='<mark>alert</mark>(<mark>document.cookie</mark>)'&gt;<mark>XSS</mark>…")
```

Comment:
I removed the `&lt;`...I believe this has to do with how the `tsvector` column handles encodings